client/src/Game.cpp

```diff
@@ -47,20 +71,15 @@ void Game::update(sf::Time deltaTime) {
     player.update(deltaTime);
 
     sf::Vector2f movement(0.f, 0.f);
-    if (playerCommands.isMIsMovingUp())
-        movement.y -= 1;
-    if (playerCommands.isMIsMovingDown())
-        movement.y += 1;
-    if (playerCommands.isMIsMovingLeft())
-        movement.x -= 1;
-    if (playerCommands.isMIsMovingRight())
-        movement.x += 1;
-
-    if(movement != sf::Vector2f(0.0f, 0.0f)) {
-        normalize(movement);
-        movement *= player.getSpeed();
-        player.move(movement * deltaTime.asSeconds());
-    }
+    if (playerCommands.isMovingUp())
```

Comment:
Ok, I will add a Player in PlayerCommands in order to move this code in PlayerCommands.